pkg/scheduler/scheduler.go

```diff
@@ -109,12 +112,28 @@ func WithBindTimeoutSeconds(bindTimeoutSeconds int64) Option {
 	}
 }
 
+// WithPodInitialBackoffDurationSeconds sets podInitialBackoffDurationSeconds for Scheduler, the default value is 1
```

Comment:
Could you elaborate the reason?

Reply:
> To avoid the comments getting outdated.

Given the odds of updating the default value is small and the other similar functions also document the default vlaue, I prefer to keep it in the comments.

Reply:
could you please take care of this?